resources/python-module-py.cc

```diff
@@ -1,30 +1,70 @@
-/**
- * @file python-module-py.cc
- * @author Maximilien Naveau (maximilien.naveau@gmail.com)
- * @license License BSD-3-Clause
- * @copyright Copyright (c) 2019, New York University and Max Planck Gesellschaft.
- * @date 2019-05-22
- * 
- * @brief This file is copied/inspired from
- * https://github.com/jrl-umi3218/jrl-cmakemodules/blob/master/dynamic_graph/python-module-py.cc
- */
+// Copyright (C) 2008-2014 LAAS-CNRS, JRL AIST-CNRS.
+//
+// This file is part of jrl-cmakemodules.
+// jrl-mathtools is free software: you can redistribute it and/or modify
+// it under the terms of the GNU Lesser General Public License as published by
+// the Free Software Foundation, either version 3 of the License, or
+// (at your option) any later version.
+//
+// jrl-mathtools is distributed in the hope that it will be useful,
+// but WITHOUT ANY WARRANTY; without even the implied warranty of
+// MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+// GNU Lesser General Public License for more details.
+// You should have received a copy of the GNU Lesser General Public License
+// along with jrl-mathtools.  If not, see <http://www.gnu.org/licenses/>.
 
 #include <Python.h>
 
-
 /**
    \brief List of python functions
 */
 static PyMethodDef functions[] = {
   {NULL, NULL, 0, NULL}        /* Sentinel */
 };
 
-PyMODINIT_FUNC
-initwrap(void)
+#if PY_MAJOR_VERSION >= 3
+static struct PyModuleDef module = {
+    PyModuleDef_HEAD_INIT,
+    "wrap",
+    NULL,
+    0,
+    functions,
+    NULL,
+    NULL,
+    NULL,
+    NULL};
+#endif
+
+#ifdef __cplusplus
+extern "C" {
+#endif
+
+#if PY_MAJOR_VERSION >= 3
+PyMODINIT_FUNC PyInit_wrap(void)
+#else
+void initwrap(void)
+#endif
 {
     PyObject *m;
 
+#if PY_MAJOR_VERSION >= 3
+    m = PyModule_Create(&module);
+#else
     m = Py_InitModule("wrap", functions);
+#endif
     if (m == NULL)
```

Comment:
According to our style guide, the `if` needs braces.  However, in case that file is a plain copy&paste from LAAS, I would rather not change it so we can more easily update in the future if needed.

Reply:
Agreed on not formatting this code. I will merge that